policy/modules/system/systemd.if

```diff
@@ -102,7 +102,7 @@ interface(`systemd_exec_systemctl',`
 	systemd_login_read_pid_files($1)
 	systemd_passwd_agent_exec($1)
 
-	dontaudit $1 self:capability net_admin;
+	dontaudit $1 self:capability { net_admin sys_ptrace };
```

Comment:
Given how much troubles there are with dontaudit rules, did you think of a diffferent approach?

Reply:
Giving every domain that execs systemctl sys_ptrace seems too excessive to me. An alternative could be to do a transition on systemctl calls (I mentioned that in the commit message), but I'm not sure about the implications... It would probably become impossible to control which domain can control which service (unless each calling domain gets its own something_sysctl_t type...)

Reply:
OK, I don't see any easy solution. Perhaps some dedicated capability?